### PR TITLE
Add words about citation to affiliated package review

### PR DESCRIPTION
**Pull request by kelle**
Adds one sentence to the Documentation section of the Affiliated Package review process. (I will also open an PR on the website to provide a link to some resources.)

### CODE REVIEW

File: affiliated/affiliated_package_review_guidelines.md
Comment (pllim) [inline affiliated/affiliated_package_review_guidelines.md:111]: Is there a guideline somewhere else on "preferred method of citation"? Even coordinated packages do not qualify under this new requirement. We should clean up our own stuff first before we impose it on others.

Comment (kelle) [reply affiliated/affiliated_package_review_guidelines.md:111]: Citation instructions should be in a CITATION file at the top most level of a repo. I just did a quick spot check and astroquery, photutils, and ccdproc all have these files.  

I will add more words to the [webpage PR](https://github.com/astropy/astropy.github.com/pull/460) with this detail.

Comment (pllim) [reply affiliated/affiliated_package_review_guidelines.md:111]: Thanks for checking but we have more coordinated packages than that. Here are the ones that do not:

* https://github.com/astropy/astropy-healpix
* https://github.com/astropy/regions
* https://github.com/astropy/reproject

Comment (pllim) [reply affiliated/affiliated_package_review_guidelines.md:111]: Also, yes, CITATION file should be low-bar enough. I was worried you were also requiring Zenodo DOI, which while is a good practice, might present higher barrier.

Comment (pllim) [reply affiliated/affiliated_package_review_guidelines.md:111]: tl;dr -- We should add CITATION files to the three coordinated packages above before merging this and astropy/astropy.github.com#460 .

**Comment by kelle** [comment]
Issues opened to request those coordinated packages add a CITATION file:
- [astropy-healpix/#167](https://github.com/astropy/astropy-healpix/issues/167)
- [regions/#409](https://github.com/astropy/regions/issues/409)
- [reproject/#262](https://github.com/astropy/regions/issues/409)

**Comment by pllim** [comment]
I cannot edit your post above, but this is an easier way to cross-ref issues across GitHub (also, the third URL was wrong):

* astropy/astropy-healpix#167
* astropy/regions#409
* astropy/reproject#262

**Comment by dhomeier** [comment]
Re: guidelines on actually _how to cite_, we certainly should not set any strict rules, but some hints on _what_ could get inside the `CITATION.x` file could perhaps be useful.
E.g. This can consist of simply providing a Zenodo link if the package has an entry (highly recommended), or the bibliography of papers describing it. Full BibTeX entries will be useful for many users publishing results based on your package.

Needs of course some good balance of minimal info and making too lengthy prescriptions; also wondering if the details are better put into the guidelines (unless both are to be kept closely sync'ed).

**Comment by kelle** [comment]
I added more words to the website. Check out astropy/astropy.github.com#460

**Review by dhomeier — APPROVED** [review]
Good to go once astropy/astropy.github.com#460 is done!

**Comment by kelle** [comment]
I vote yes but am not sure of who should push the button. @pllim ?

**Comment by pllim** [comment]
Sure, though I see healpix and reproject are still lagging. I cannot merge here.

**Comment by kelle** [comment]
I am happy with the discussion having started in both places. Merging.

**Pull request merged**
